src/main/java/UI/CartPage.java

```diff
@@ -23,9 +24,9 @@ public class CartPage extends Page implements ActionListener {
 
 
     public CartPage() {
-        super(User.getCurrentUser().getUsername() + "'s Cart");
+        super(Main.getCurrentUser().getUsername() + "'s Cart");
```

Comment:
Maybe instead of the User's username's Cart, you can use their name?

Reply:
what do you mean by this? we only store username, so we can't display their actual name. furthermore, i'm ok with having "Cart" in the title. i think the way mark is doing it now is good.